### PR TITLE
DOCSP-31491: update synonym file

### PR DESCRIPTION
**Pull request by sarahemlin**
@casthewiz -

I added synonyms for searches with zero results and a few others I've been collecting.  If this process works, let me know who else could approve PRs for the synonym file going forward.

Thanks!
Sarah

### CODE REVIEW

**Review by casthewiz — APPROVED** [review]
This LGTM - in the future, feel free to request @seungpark or @branberry as reviewers also.

(And feel free to merge on a single approval).

We'll be able to observe the results in staging post-merge, and make additional changes if we need to.

File: resources/synonyms.csv [hunk 1]
```diff
@@ -15,4 +15,41 @@ iot,internet of things,
 python async,motor,
 node,nodejs,node.js
 shell,mongosh,
-download,descargar,
\ No newline at end of file
+download,descargar,
+operator,$$,dollar sign,
```
Comment (seungpark) [inline resources/synonyms.csv:19]:
```suggestion
operator,$,$$,dollar sign,

```

we can remove line 21 after

File: resources/synonyms.csv [hunk 2]
```diff
@@ -15,4 +15,41 @@ iot,internet of things,
...
+operator,$,
+$elemMatch,$elementMatch,elementMatch,match element,
+$arrayElemAt,$arrayElementAt,arrayElementAt,$elemAt,
+$inc,$incr,
+autocomplete,search as you go,search-as-you-type,
+js,javascript,
+readonly,read only,
+config,configuration,
+wiredTiger,wired tiger,
+mongosh,shell,mongo shell,mongodb shell,
```
Comment (seungpark) [inline resources/synonyms.csv:30]: similar here with line 17

File: resources/synonyms.csv [hunk 3]
```diff
@@ -15,4 +15,41 @@ iot,internet of things,
...
+one-to-many,1:many,
+expr,expression,
+CAFile,certificate authority file,
+crl,certificate revocation list,
+sqrt,square root,
+trunc,truncate,
+gcp,Google Cloud Platform,
+cond,conditional,
+collMod,modify collection,
+ne,not equal,
```
Comment (seungpark) [inline resources/synonyms.csv:50]: this can be merged with line 20

**Review by seungpark — CHANGES_REQUESTED** [review]
caught a few minor repeated synonyms! PTAL @sarahemlin

File: resources/synonyms.csv [hunk 4]
```diff
@@ -15,4 +15,42 @@ iot,internet of things,
 python async,motor,
 node,nodejs,node.js
 shell,mongosh,
-download,descargar,
\ No newline at end of file
+download,descargar,
+operator,$,$$,dollar sign,
+
+$ne,$neq,not equal,
```
Comment (seungpark) [inline resources/synonyms.csv:21]:
```suggestion
$ne,$neq,not equal,ne,not equal,
```

File: resources/synonyms.csv [hunk 5]
```diff
@@ -15,4 +15,42 @@ iot,internet of things,
...
+one-to-many,1:many,
+expr,expression,
+CAFile,certificate authority file,
+crl,certificate revocation list,
+sqrt,square root,
+trunc,truncate,
+gcp,Google Cloud Platform,
+cond,conditional,
+collMod,modify collection,
+ne,not equal,
```
Comment (seungpark) [inline resources/synonyms.csv:51]:
```suggestion
```

File: resources/synonyms.csv [hunk 6]
```diff
@@ -15,4 +15,42 @@ iot,internet of things,
 python async,motor,
 node,nodejs,node.js
 shell,mongosh,
```
Comment (seungpark) [inline resources/synonyms.csv:17]:
```suggestion
shell,mongosh,mongo shell,mongodb shell,
```

File: resources/synonyms.csv [hunk 7]
```diff
@@ -15,4 +15,42 @@ iot,internet of things,
...
+operator,$,
+$elemMatch,$elementMatch,elementMatch,match element,
+$arrayElemAt,$arrayElementAt,arrayElementAt,$elemAt,
+$inc,$incr,
+autocomplete,search as you go,search-as-you-type,
+js,javascript,
+readonly,read only,
+config,configuration,
+wiredTiger,wired tiger,
+mongosh,shell,mongo shell,mongodb shell,
```
Comment (seungpark) [inline resources/synonyms.csv:31]:
```suggestion

```

File: resources/synonyms.csv [hunk 8]
```diff
@@ -15,4 +15,42 @@ iot,internet of things,
 python async,motor,
 node,nodejs,node.js
 shell,mongosh,
-download,descargar,
\ No newline at end of file
+download,descargar,
+operator,$,$$,dollar sign,
+
```
Comment (seungpark) [inline resources/synonyms.csv:20]:
```suggestion

```

**Review by seungpark — CHANGES_REQUESTED** [review]
@sarahemlin i left some suggestions. the row numbers were getting mixed up with commits

**Review by seungpark — APPROVED** [review]
LGTM 👍

**Comment by seungpark** [comment]
@sarahemlin i deleted an old remnant NULL entry. i think this may have been a remnant of previous commits. will drop in a ticket to refresh the table to prevent this error. we're good to merge!

**Pull request merged**